src/main/java/lithium/openstud/driver/core/models/ExamReservation.java

```diff
@@ -24,6 +25,10 @@ public class ExamReservation {
     private String ssd;
     private String module;
 
+    private JSONArray attendingModesList;
```

Comment:
Instead of relying on JSONArray from the jsoup lib to expose the information I think it would be better to use the Java stdlib (List<String>)